### ISSUE TITLE
.asEventStream('...', function(e){})  will always returns the event

### ISSUE THREAD

**Issue by BlackYoup** [baconjs/bacon.js#648]
The function will always return the event instead of a the value the user wants to return

```
$('body').asEventStream('click', e => 'body clicked !').log();
```
prints the event, not `` body clicked !``

It came with version ``0.7.76``, still in ``0.7.77``

**Comment by raimohanska** [comment]
Fixed in 0.7.78. Thanks for the report! This has been broken in the coffeescript->es6 transformation, most probably.

**Comment by BlackYoup** [comment]
Thanks :)
